### PR TITLE
Miniconda3 updates

### PR DESCRIPTION
**Pull request by Wolf1098**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs) (edit have now, couldn't before post) 
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----
Version bump of anaconda.miniconda3, as well as added user/machine scopes, and included x86 installer.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/28142)

### CODE REVIEW

File: manifests/a/Anaconda/Miniconda3/py39_4.10.3/Anaconda.Miniconda3.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/microsoft/winget-cli/master/schemas/JSON/manifests/v1.0.0/manifest.defaultLocale.1.0.0.json
+PackageIdentifier: Anaconda.Miniconda3
+Moniker: miniconda3
+PackageVersion: py39_4.10.3
+PackageLocale: en-US
+Publisher: Anaconda
```
Comment (OfficialEsco) [inline manifests/a/Anaconda/Miniconda3/py39_4.10.3/Anaconda.Miniconda3.locale.en-US.yaml:6]:
```suggestion
Publisher: Anaconda, Inc.
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
This needs to match whats show in `A`dd or `R`emove `P`rograms

File: manifests/a/Anaconda/Miniconda3/py39_4.10.3/Anaconda.Miniconda3.locale.en-US.yaml (hunk 1 above)
Comment (Wolf1098) [reply manifests/a/Anaconda/Miniconda3/py39_4.10.3/Anaconda.Miniconda3.locale.en-US.yaml]: Older version of package has same field for publisher. The old version is also installed before the newer versions prepended with py*.
https://github.com/microsoft/winget-pkgs/blob/master/manifests/a/Anaconda/Miniconda3/4.8.3/Anaconda.Miniconda3.yaml

Comment (OfficialEsco) [reply manifests/a/Anaconda/Miniconda3/py39_4.10.3/Anaconda.Miniconda3.locale.en-US.yaml]: I see, @ItzLevvie corrected py39_4.9.2 which fixed the matching but left 4.8.3 which broke the version selection..
Would you mind updating or deleting that version?

**Review by OfficialEsco — DISMISSED** [review]

File: manifests/a/Anaconda/Miniconda3/py39_4.10.3/Anaconda.Miniconda3.locale.en-US.yaml (hunk 1 above)
Comment (Wolf1098) [reply manifests/a/Anaconda/Miniconda3/py39_4.10.3/Anaconda.Miniconda3.locale.en-US.yaml]: Alright, changes made

**Comment by OfficialEsco** [comment]
Oof, you can't edit a Manifest with a different version in the same PR, you need to do 2 PRs.
Sorry for the confusion

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Only one package per PR.

**Comment by Wolf1098** [comment]
Does the repo support rebasing pull requests or should I just commit a reversal :)

**Review by OfficialEsco — APPROVED** [review]

**Comment by Wolf1098** [comment]
So for future reference, that works, but seems a bit bodge, as commits get orphaned, but the pull request tracks the history with parents.

**Comment by OfficialEsco** [comment]
Leave the PR be, the Administrators need to add a .validation file for the domain

**Pull request merged**